src/Dapr.Client/DaprClientGrpc.cs

```diff
@@ -88,19 +119,20 @@ private async Task MakePublishRequest<TContent>(string topicName, TContent conte
             var envelope = new Autogenerated.InvokeBindingRequest()
```

Comment:
Could you please add null check and throw for name and operation
ArgumentVerifier.ThrowIfNullOrEmpty(operation, nameof(operation));.